quaternions/utils.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+
+
+def covariance_matrix_from_angles(angles_list):
+    """
+    Computes covariance matrix from 3d vectors list
+    """
+    covariance_matrix = np.zeros((3, 3))
+    for angles in angles_list:
+        covariance_matrix += np.outer(angles.T, angles)
```

Comment:
I guess `angles_list` is a list of NumPy arrays, right?

Reply:
yes